.github/workflows/html-app/publish.yml

```diff
@@ -54,10 +54,10 @@ jobs:
         id: build-and-push
         uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
         with:
-          context: .
+          context: html-app
+          file: html-app/Dockerfile
           push: true
-#          tags: ${{ steps.meta.outputs.tags }}
-          tags: ghcr.io/eclipse-pass/${{ env.IMAGE_NAME }}:1.0
+          tags: ${{ steps.meta.outputs.tags }}
```

Comment:
Was the `# tags: ${{ steps.meta.outputs.tags }}` commented out now commented in originally not working, but works now ?

Reply:
Yes. I think it was having trouble because it was the first PR and couldn't figure out how to generate a valid tag using the PR number.